tools/bundle-builder/docs/index.js

```diff
@@ -263,6 +263,11 @@ function buildSite_html() {
     .pipe(gulp.dest('dist/docs/'));
 }
 
+function copySiteWorkflowIcons() {
+  return gulp.src('node_modules/@adobe/spectrum-css-workflow-icons/dist/spectrum-icons.svg')
```

Comment:
Doesn't `@adobe/spectrum-css-workflow-icons` support `require.resolve()`? If so, this can be changed to:

```
return gulp.src(path.join(path.dirname(require.resolve('@adobe/spectrum-css-workflow-icons')), 'dist', 'spectrum-icons.svg'))
```